test/e2e/helpers/driver.ts

```diff
@@ -120,6 +115,24 @@ export async function isDriverInstanceClosed(driver: WebDriver, handleToSwitchBa
   return correctException
 }
 
+// This is not something we want to do. We should ONLY use this on ECONNREFUSED exceptions in tests that use the popout functionality, not to accommodate for UI flakiness.
+// This is because selenium randomly crashes due to our auto open/close functionality, however we still want to test it.
```

Comment:
It is flaky enough behaviour that if it fails a test multiple times in a row we can say quite confidently there is a genuine bug. The driver crash happens approx 1/5 runs.

Reply:
Want it on record that I do not like this fix - at all. However, the driver crashing leaves stale instances of driver handles around and it happens spontaneously after saying confirm/reject on a popout and the popout auto closes. Have tried multiple solutions but can confidently say this is not our bug.